Userland/Applications/MailSettings/MailSettingsWindow.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021, The SerenityOS developers
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGUI/TextEditor.h>
+#include <LibGUI/Window.h>
+
+class MailSettingsWindow final : public GUI::Window {
+    C_OBJECT(MailSettingsWindow)
+public:
+    virtual ~MailSettingsWindow() override;
+
+private:
+    MailSettingsWindow();
+
+    void reset_default_values();
+    void write_values();
+
+    String server = "";
+    int port = 0;
+    bool tls = false;
+    String email = "";
+    RefPtr<Core::ConfigFile> config;
```

Comment:
Names for private fields should have an `m_` prefix. Also, please use `{ ... }` to initialize these to match the style of the rest of the codebase.